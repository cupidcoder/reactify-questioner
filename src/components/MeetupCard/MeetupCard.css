.MeetupCard {
  box-shadow: 6px 3px 11px 3px rgba(128,128,128,0.61);
  border-radius: 3px;
  padding: 1.5rem;
  flex: 0 1 calc(24% - 1rem);
  margin-bottom: 1.5rem;
  cursor: pointer;
  transition: 0.5s transform;
}

.MeetupCard:hover {
  transform: scale(1.03);
  box-shadow: 2px 1px 11px 1px rgba(128,128,128,0.61);
}

.MeetupCardContent h3 {
  margin-bottom: 1rem;
}

.MeetupCardContent p {
  margin-bottom: 0.5rem;
}

.MeetupCardContent p i {
  margin-right: 1rem;
  color: #305b8c;
}

.Link:link, .Link:visited {
  color: #000000;
  text-decoration: none;
}

.Link:hover, .Link:active {
  color: #000000;
  text-decoration: none;
}

@media screen and (max-width: 415px) {
  .MeetupCard {
    margin: 1rem auto;
    flex: 0 1 calc(49% - 1rem);
  }
}
