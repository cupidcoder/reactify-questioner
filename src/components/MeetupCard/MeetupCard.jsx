import React from 'react';
import { Link } from 'react-router-dom';
import style from './MeetupCard.css';

const MeetupCard = ({
  id, name, description, location
}) => (
  <Link to={`meetups/page/${id}`} className={style.Link}>
    <div className={style.MeetupCard}>
      <div className={style.MeetupCardContent}>
        <h3>{name}</h3>
        <p>{description}</p>
        <p>
          <i className='fas fa-map-marker-alt' />
          {location}
        </p>
      </div>
    </div>
  </Link>
);

export default MeetupCard;
