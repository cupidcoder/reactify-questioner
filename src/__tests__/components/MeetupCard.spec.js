import React from 'react';
import { shallow } from 'enzyme';
import MeetupCard from '../../components/MeetupCard/MeetupCard';

describe('MeetupCard', () => {
  it('Should correctly save snapshot of the MeetupCard', () => {
    const wrapper = shallow(<MeetupCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
