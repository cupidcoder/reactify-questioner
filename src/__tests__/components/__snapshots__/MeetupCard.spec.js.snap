// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupCard Should correctly save snapshot of the MeetupCard 1`] = `ShallowWrapper {}`;
